lib/procore/requestable.rb

```diff
@@ -19,17 +19,19 @@ module Requestable
     #
     # @return [Response]
     def get(path, query: {}, options: {})
+      full_path = File.join(base_api_path, path).to_s
```

Comment:
seems like these are all the same? Add a method?

Reply:
nvm, what @nhessler said 😂

Reply:
Added that in 👍